f3/powertable.go

```diff
@@ -36,10 +36,7 @@ func NewPowerTable(entries []PowerEntry) *PowerTable {
 		lookup[entry.ID] = i
 		total.Add(total, entry.Power)
 	}
-
-	if len(entries) != len(lookup) {
-		panic("duplicate power entries")
-	}
```

Comment:
This one should return an error instead. It will be called directly by the client user. We need to protect against duplicates, but they need to be able to catch the error.

Reply:
Bc of line 33 of the same function this is impossible though.